components/dashboard/donation/editDonationModalBody.js

```diff
@@ -188,6 +188,8 @@ const EditDonationModalBody = ({ donationAmount, isNewDonor, onClose }) => {
                     defaultValue={donationAmount}
                     min={5}
                     max={10000}
+                    onChange={handleNewAmount}
+                    clampValueOnBlur={false}
```

Comment:
had to override this to false since when it's true, it'd throw a JS error from internals of chakra for some reason i didn't have time to dig into